builder/builder.go

```diff
@@ -55,7 +55,7 @@ type Opt struct {
 	Console                               conslogging.ConsoleLogger
 	Verbose                               bool
 	Attachables                           []session.Attachable
-	Enttlmnts                             []entitlements.Entitlement
+	Entitlements                          []entitlements.Entitlement
```

Comment:
Sometimes capital letters hint at public apis that can't be changed. In general, assuming this isn't a public api, there didn't seem to be a reason not to use the full word, especially give that it's only one character longer than `Attachables`...